src/landlab/components/overland_flow/generate_overland_flow_implicit_kinwave.py

```diff
@@ -211,11 +211,15 @@ def __init__(
         ----------
         grid : ModelGrid
             Landlab ModelGrid object
-        runoff_rate : float, optional (defaults to 1 mm/hr)
-            Precipitation rate, mm/hr. The value provided is divided by
-            3600000.0.
-        roughness : float, defaults to 0.01
-            Manning roughness coefficient; units depend on depth_exp.
+        runoff_rate : array_like of float or str (defaults to 1 mm/hr)
```

Comment:
For both `runoff_rate` and `roughness`, how about:
```
runoff_rate : str or array_like of float
```
This makes it clear that it's not an array of either floats or strings but, rather, a string or an array of floats.